client/my-sites/plans-grid/components/plan-type-selector/components/interval-type-toggle.tsx

```diff
@@ -29,14 +30,15 @@ export const IntervalTypeToggle: React.FunctionComponent< IntervalTypeProps > =
 		'is-signup': isInSignup,
 	} );
 	const popupIsVisible = Boolean( intervalType === 'monthly' && props.plans.length );
-	const maxDiscount = useMaxDiscount( props.plans, usePricingMetaForGridPlans, selectedSiteId );
+	const maxDiscount = useMaxDiscount( props.plans, usePricingMetaForGridPlans, useCheckPlanAvailabilityForPurchase, selectedSiteId );
```

Comment:
@jdc91 it's almost always better to create single-parameter (object) functions vs this wide enumeration of props as separate arguments. It's easier to maintain & extend that way, and more consistent with everything else in React.

Reply:
Least to say just plainly more usable. 
Not sure when would `foo( undefined, undefined, 123, undefined, 234 )` be preferred over `foo( { bar: 123, baz: 234 } )` :-)